plugins/Satellites/src/gui/SatellitesDialog.cpp

```diff
@@ -1240,7 +1247,7 @@ void SatellitesDialog::setRightSideToROMode()
 	ui->periodLineEdit->setEnabled(false);
 	ui->periodLineEdit->setText(QString());
 
-	// set default
+	// set default // XXX copy-pasta
 	buttonMarkerColor = QColor(QColor::fromRgbF(0.4, 0.4, 0.4));
```

Comment:
It's actually possible to remove the wrapper. Just 

`buttonMarkerColor = QColor::fromRgbF(0.4, 0.4, 0.4);
`
etc.